testsuite/src/main.rs

```diff
@@ -10,12 +12,37 @@ fn main() {
         eprintln!("Expecting two arguments: <w3c_rdf_tests_base_path> <manifest_url>");
         return;
     }
+    let mut parse_func: fn(&str, &Path) -> Result<OwnedDataset, Box<dyn Error>> =
+        parse_w3c_rdf_test_file;
+    let mut manifest_url: &str = &args[2];
+
+    if args[2] == "ntriples" || args[2] == "nt" {
+        manifest_url = "http://w3c.github.io/rdf-tests/ntriples/manifest.ttl";
+    }
+    if args[2] == "nquads" || args[2] == "nq" {
+        manifest_url = "http://w3c.github.io/rdf-tests/nquads/manifest.ttl";
+    }
+    if args[2] == "turtle" || args[2] == "ttl" {
+        manifest_url = "http://w3c.github.io/rdf-tests/turtle/manifest.ttl";
+    }
+    if args[2] == "rdf-xml" || args[2] == "rdf" {
+        manifest_url = "http://w3c.github.io/rdf-tests/rdf-xml/manifest.ttl";
+    }
+    if args[2] == "trig" {
+        manifest_url = "http://w3c.github.io/rdf-tests/trig/manifest.ttl";
+    }
```

Comment:
Is this big set of `if` actually used anywhere? If yes, using a `match` might be much nicer.